src/App.js

```diff
@@ -1,102 +1,105 @@
 import React, { Component } from 'react';
-import './App.css';
-import Navbar from './Components/Navbar';
-import LandingPage from './Components/LandingPage';
-import SearchPage from './Components/SearchPage';
+import './styles/Main.scss';
```

Comment:
Main contains all references to scss files

Reply:
No it wasn't. Good catch

Reply:
App.css have now been moved to styles, added to Main.scss and renamed to App.scss